src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/X509Certificates/X509CertificateKeyAccessors.cs

```diff
@@ -395,6 +395,86 @@ public static X509Certificate2 CopyWithPrivateKey(this X509Certificate2 certific
 #endif
         }
 
+        /// <summary>
+        ///   Gets the <see cref="CompositeMLDsa"/> public key from this certificate.
+        /// </summary>
+        /// <param name="certificate">
+        ///   The X.509 certificate that contains the public key.
+        /// </param>
+        /// <returns>
+        ///   The public key, or <see langword="null"/> if this certificate does not have a Composite ML-DSA public key.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="certificate"/> is <see langword="null"/>.
+        /// </exception>
+        /// <exception cref="PlatformNotSupportedException">
+        ///   The certificate has a Composite ML-DSA public key, but the platform does not support Composite ML-DSA.
+        /// </exception>
+        /// <exception cref="CryptographicException">
+        ///   The public key was invalid, or otherwise could not be imported.
+        /// </exception>
+        [Experimental(Experimentals.PostQuantumCryptographyDiagId, UrlFormat = Experimentals.SharedUrlFormat)]
+        public static CompositeMLDsa? GetCompositeMLDsaPublicKey(this X509Certificate2 certificate)
```

Comment:
This is implementable like ML-KEMs is. We don't need platform support for handling SPKIs.